portfolio/src/main/webapp/scripts/index.js

```diff
@@ -1,13 +1,23 @@
 import { updateRepos } from './github/index.js';
 import { updateVideos } from './youtube/index.js';
 import { updatePosts } from './medium/index.js';
-import { addComments } from './api/index.js';
+import { updateComments, addComment } from './api/index.js';
```

Comment:
nit: probably makes sense to alphabetize this

Reply:
Done! Thanks 👯